src/handlers/index.py

```diff
@@ -33,6 +34,23 @@ def handler(event, context):
 
     return kubectl(event["commands"])
 
+def parse_json(json_str: str) -> Optional[Dict]:
+    try:
+        return json.loads(json_str)
+    except json.JSONDecodeError:
+        json_objects = []
+
+        try:
+            for line in json_str.strip().split('\n'):
+                try:
+                    json_objects.append(json.loads(line))
+                except json.JSONDecodeError as e:
+                    logger.error(f"Failed to parse line: {line}, error: {e}")
+
+        except (TypeError, AttributeError) as e:
+            logger.error("Failed to parse json: %s" % e)
+
+        return json_objects
```

Comment:
I'm guessing this function came about as a result of some inconsistent log data. Might be worth adding in some unit tests to handle each edge case as it arises and help guarantee regressions don't come in with future updates.